webpack.config.js

```diff
@@ -113,7 +113,8 @@ module.exports = {
     alias: {
       css: path.join(__dirname, "/troposphere/static/css/"),
       images: path.join(__dirname, "/troposphere/static/images/"),
-      highcharts: "highcharts-commonjs"
+      highcharts: "highcharts-commonjs",
+      react: path.join(__dirname, "/node_modules/react")
```

Comment:
When you run `npm install` in a project, it installs peerDependencies. The only time they aren't installed is when you pull it down from NPM itself. So if you're developing `cyverse-ui` locally, `react` will be in its `node_modules` from running `npm install` to develop it. I believe that's where the conflict is coming from - the `import react from 'react'` statement within `cyverse-ui` is resolving to *its* React package, and Troposphere is resolving to *its* React package. Thus the conflict.

Or I could be totally wrong : ) Admittedly I don't totally understand the impact of @cdosborn's "fallback" find and how that interacts with the standard behavior of packages to use their own dependencies. I guess I'll need to carve out time to experiment with it/code dive at some point.